src/components/ErrorMessage.tsx

```diff
@@ -1,39 +1,72 @@
 import { BodyText } from "components/Text"
 import { FC } from "react"
-import { StyleProp, TextStyle, View, ViewStyle } from "react-native"
-import { Icon } from "./Icon"
+import { StyleProp, View, TextStyle } from "react-native"
+import { Canvas, Group, ImageSVG, useSVG } from "@shopify/react-native-skia"
+import polifemoIcon from "assets/polifemo/happy.svg" // <-- NON è quella giusta!!
+import { usePalette } from "utils/colors"
+import { useTranslation } from "react-i18next"
 
 interface ErrorMessageProps {
   message: string
-  styleView?: StyleProp<ViewStyle>
   styleMessage?: StyleProp<TextStyle>
-  image?: number
 }
 
 /**
  * This component displays a message if an error occurs.
  */
 export const ErrorMessage: FC<ErrorMessageProps> = props => {
+  const polifemoSVG = useSVG(polifemoIcon)
+
+  const { primary } = usePalette()
+
+  const { t } = useTranslation("freeClass")
+
   return (
-    <View style={[{ flex: 1, marginHorizontal: 20 }, props.styleView]}>
-      {props.image ? (
-        <Icon style={{ alignSelf: "center" }} source={props.image} />
-      ) : undefined}
-      <BodyText
-        style={[
-          {
-            flex: 1,
-            color: "red", //TBC
-            fontWeight: "400", //TBC
-            fontSize: 30, //TBC
-            alignSelf: "center",
-            textAlign: "center",
-          },
-          props.styleMessage,
-        ]}
+    <View
+      style={{
+        justifyContent: "center",
+        flex: 0.8,
+      }}
+    >
+      <View
+        style={{
+          justifyContent: "space-evenly",
+          flex: 0.9,
+        }}
       >
-        {props.message}
-      </BodyText>
+        <Canvas
+          style={{
+            height: 192,
+            width: 97.5,
+            alignSelf: "center",
+          }}
+        >
+          {polifemoSVG && (
+            <Group transform={[{ scale: 2.73 }]}>
+              <ImageSVG
+                svg={polifemoSVG}
+                x={-9.2}
+                y={-29.5}
+                width={63}
+                height={128}
+              />
+            </Group>
+          )}
+        </Canvas>
+        <BodyText
+          style={[
+            {
+              color: primary,
+              fontWeight: "400",
+              fontSize: 16,
+              textAlign: "center",
+            },
+            props.styleMessage,
+          ]}
+        >
+          {t("freeClass_error") + props.message}
+        </BodyText>
```

Comment:
questa cosa qui dovrebbe essere diversa, ci dovrebbe essere sopra un po' più grande la scritta "Ops!" normale e "ERRORE" in grassetto, poi _sotto_ invece il messaggio, tre stili separati